docs/content/en/docs/pipeline-stages/builders/ko.md

```diff
@@ -216,27 +217,115 @@ Useful tips for existing `ko` users:
   file. If you already configure your base images using this file, you do not
   need to repeat this configuration in `skaffold.yaml`.
 
-- When you use the Skaffold ko builder, Skaffold takes care of replacing the
-  image placeholder name in your Kubernetes manifest files using its
-  [render]({{< relref "/docs/pipeline-stages" >}}) functionality.
-
-- The ko builder supports image name placeholders that consist of the
-  `ko://` prefix, followed by the Go import path to the main package.
-  This means that Skaffold should work with existing Kubernetes manifest
-  files that use this image name placeholder format.
-
-  If you previously rendered Kubernetes manifest using `ko`, e.g.:
+- Generate a starter `skaffold.yaml` file by searching your existing manifests
+  for image references starting with `ko://` by using this snippet:
 
   ```shell
-  ko resolve --filename k8s/*.yaml > out.yaml
+  cat << EOF > skaffold.yaml
+  apiVersion: skaffold/v2beta26
+  kind: Config
+  build:
+    artifacts:
+  EOF
+  grep -rho "ko://$(go list -m)[^\"]*" ./config/ | sort | uniq | xargs -Iimg echo -e "  - image: img\n    ko: {}" >> skaffold.yaml
```

Comment:
We need to teach `skaffold init` how to recognize a Ko set up.  Can you mark this snippet as being a temporary measure?

Reply:
Good point, and `init` support is in our list of future features. I've added a note as you suggest for now.